src/common-lib/urls/urls.ts

```diff
@@ -585,6 +591,12 @@ export const OAK_PAGES: {
     configType: "internal",
     pageType: "curriculum-landing-page",
   }),
+  "early-release-units-page": createOakPageConfig({
+    pathPattern: "/beta/teachers-2023/early-release-units",
```

Comment:
this has also been done [in my branch](https://github.com/oaknational/Oak-Web-Application/pull/1908/files#diff-deaf037896e78be30b3fdf06dec81c6f72870d073f5f1fabf655731ff7e62c90) but moved out of `/beta` - shall we align them before merging?